sources/documents/2_RTB/Verbali/Esterni/VerbaleEsterno_2024-05-06/capitolo-riunione.tex

```diff
@@ -0,0 +1,59 @@
+\section{Riunione}
+\subsection{Ordine del giorno}
+\begin{itemize}
+	\item Discutere delle tecnologie per lo sviluppo della web app;
+	\item Esporre studio preliminare delle tecnologie lato back-end;
+	\item Valutare l'ampliamento dei casi d'uso;
+\end{itemize}
+
+\subsection{Argomenti e temi dell'incontro}
+
+\subsubsection{Tecnologie per lo sviluppo della web app}
+
+\textbf{Domanda:} Tra le diverse tecnologie solitamente indicate per lo sviluppo di un'applicazione web (quali Node.js, Flask etc.), abbiamo individuato il \glossario{framework} \glossario{Streamlit}. 
```

Comment:
Tra le diverse tecnologie solitamente indicate per lo sviluppo di un'applicazione web (quali Django, Flask, React, Vue.js), abbiamo individuato il \glossario{framework} \glossario{Streamlit}. 
Il \glossario{framework} in questione permette di mantenere il linguaggio \glossario{Python} per sviluppare l'interfaccia grafica, seppur conceda meno spazio alla personalizzazione. 
Secondo lei, potrebbe essere una scelta opportuna?